web/src/main/resources/openmrs-servlet.xml

```diff
@@ -54,6 +54,8 @@
 	
 	<bean id="contentNegotiationManager"
 		  class="org.springframework.web.accept.ContentNegotiationManagerFactoryBean">
+		  <property name="favorPathExtension" value="false"/>
```

Comment:
What were you trying to do here, what is the importance of that property name, did you investigate where it were called, did you run it successfully

Reply:
@sherrif10 hei sharif kindly first get to know the requirements of the ticket before commenting

Reply:
@herbert24 read my comments very well please